1d-life/src/1d-life.js

```diff
@@ -41,7 +41,17 @@
   function getNewVal(lifeState, x) {
     // !!! IMPLEMENT ME
 
-    return 0; // instead of this
+    //return Math.round(Math.random()); // just testing 
```

Comment:
lolololololol, your commit made me smile pretty big.  Very happy you got that moment : ) 

also great solution obviously